api/core/v1alpha2/vmcondition/condition.go

```diff
@@ -86,4 +87,7 @@ const (
 	ReasonFilesystemReady    Reason = "Ready"
 	ReasonFilesystemFrozen   Reason = "Frozen"
 	ReasonFilesystemNotReady Reason = "NotReady"
+
+	ReasonChosenForSnapshotting  Reason = "WaitingForTheSnapshotting"
```

Comment:
It is not obvious what we are waiting for. Is it the message about waiting for start or wait until snapshot ends?

WaitingForTheSnapshotToStart seems more accurate.